CRM/Events/Upgrader.php

```diff
@@ -29,4 +29,55 @@ class CRM_Events_Upgrader extends CRM_Extension_Upgrader_Base
     public function install()
     {
     }
+
+    public function enable(){
+
+      $overlap_text = 'Hallo Seminarverwaltung,
+
+ein/e Seminarportal-User*in hat versehentlich zwei parallel stattfindende BFD-Seminare gebucht. Bitte überprüfe die Buchungen.
+
+Freiwillige*r: {contact.display_name}
+
+Seminar 1: {$eventOneTitle}, {$eventOneStart|crmDate:"Datetime"} - {$eventOneEnd|crmDate:"Datetime"}
+Seminar 2: {$eventTwoTitle}, {$eventTwoStart|crmDate:"Datetime"} - {$eventTwoEnd|crmDate:"Datetime"}
+
+Link zum CiviCRM-Kontakt: {domain.base_url}civicrm/contact/view?reset=1&cid={contact.id}
+
+Dies ist eine automatisch generierte Nachricht.';
+
+      $overlap_html = '<p>Hallo Seminarverwaltung,</p>
+
+<p>ein/e Seminarportal-User*in hat versehentlich zwei parallel stattfindende BFD-Seminare gebucht. Bitte überprüfe die Buchungen.</p>
+
+<p>Freiwillige*r: <a href="{domain.base_url}civicrm/contact/view?reset=?cid={contact.id}">{contact.display_name}</a></p>
+
+<p>Seminar 1: {$eventOneTitle}, {$eventOneStart|crmDate:"Datetime"} - {$eventOneEnd|crmDate:"Datetime"}<br />
+Seminar 2: {$eventTwoTitle}, {$eventTwoStart|crmDate:"Datetime"} - {$eventTwoEnd|crmDate:"Datetime"}</p>
+
+<p>Dies ist eine automatisch generierte Nachricht.</p>';
```

Comment:
I'll have to agree with @bjendres this time :wink: - let's put this into a `.txt` and a `.tpl` file and be read from there.